change/@fluentui-react-slider-2fad34fd-35cd-408f-b940-e8d8b1e0c817.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32051&commitId=52b8011b7ef0c1c2811d863e03bc5b558e7fe52a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|16| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|1141| Changed|
|Drawer.overlay drawer full.chromium.png|1147| Changed|

</details>


  <details><summary><b>Field</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.Slider.chromium.png|279| Changed|

</details>


  <details><summary><b>Slider Converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider Converged.Disabled High Contrast.chromium.png|122| Changed|
|Slider Converged.Disabled Vertical.chromium.png|44| Changed|
|Slider Converged.Disabled Dark Mode.chromium.png|34| Changed|
|Slider Converged.Horizontal 0 RTL.chromium.png|139| Changed|
|Slider Converged.Root Dark Mode.chromium.png|64| Changed|
|Slider Converged.Disabled.chromium.png|44| Changed|
|Slider Converged.Horizontal - 0%.chromium.png|139| Changed|
|Slider Converged.Horizontal - 100%.chromium.png|139| Changed|
|Slider Converged.Root RTL.chromium.png|92| Changed|
|Slider Converged.Root High Contrast.chromium.png|155| Changed|
|Slider Converged.Horizontal 100 RTL.chromium.png|139| Changed|
|Slider Converged.Vertical 0 RTL.chromium.png|140| Changed|
|Slider Converged.Vertical 100 RTL.chromium.png|140| Changed|
|Slider Converged.Vertical - 0%.chromium.png|140| Changed|
|Slider Converged.Root.chromium.png|92| Changed|
|Slider Converged.Vertical - 100%.chromium.png|140| Changed|
|Slider Converged.Vertical RTL.chromium.png|92| Changed|
|Slider Converged.Vertical.chromium.png|92| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>